chore: Add repository conventions and basic configurations. (#3)
lambdaclass/lambda_ethereum_consensus: .github/pull_request_template.md

**Motivation**

> <!-- Why is this PR exists? What are the goals of the pull request? -->

Reviewer: Nit
```suggestion
<!-- Why does this pull request exist? What are its goals? -->
```

**Description**

> <!-- A clear and concise general description of the changes of this PR commits -->

Reviewer: Nit

```suggestion
<!-- A clear and concise general description of the changes this PR introduces -->
```
